### DISCUSSION TITLE
`GetDisplayNameForHitResult` can not be localised

### DISCUSSION THREAD

**Discussion by Flutterish** [ppy/osu#19630] [Ideas]
GetDisplayNameForHitResult returns a string, not a localisable string. This means rulesets with custom hit result names cannot be fully localised.

**Comment by Flutterish** [comment]
Same thing goes for mods. Is there a list of things that cannot be localised yet for rulesets?

**Reply by Flutterish** [reply]
I would like to add: SettingsSource does not have a localisable variant

**Comment by peppy** [comment]
It needs to be updated to use `GetLocalisableDescription` as per everywhere else.

**Reply by Flutterish** [reply]
Which of the points does this refer to? HitResult isnt a different enum per ruleset as far as Im aware
